data/bhattabhi013.json

```
@@ -1,68 +1,104 @@
 {
-  "name": "Abhishek Bhatt",
-  "displayStatsPublic": true,
-  "type": "personal",
-  "bio": "Full-Stack developer | Flutter | NodeJs | Angular",
-
-  "tags": [
-    "flutter",
-    "nodejs",
-    "angular",
-    "javascript",
-    "typescript",
-    "open-source"
-  ],
-  "socials": [
-    { "icon": "FaTwitter", "url": "https://twitter.com/AbhishekBtwt" },
-    { "icon": "FaGithub", "url": "https://github.com/bhattabhi013" },
-    { "icon": "FaInstagram", "url": "https://www.instagram.com/bwith_abhi" }
-  ],
-  "links": [
-    {
-      "name": "Follow me on GitHub",
-      "url": "https://github.com/bhattabhi013",
-      "icon": "FaGithub"
-    },
-    {
-      "name": "Check out my Portfolio",
-      "url": "https://bhattabhi013.github.io./",
-      "icon": "FaLink"
-    },
-    {
-      "name": "Follow me on Twitter",
-      "url": "https://twitter.com/AbhishekBtwt",
-      "icon": "FaTwitter"
-    },
-    {
-      "name": "Connect with me on Linkedin",
-      "url": "https://www.linkedin.com/in/bhatt-abhi/",
-      "icon": "FaLinkedin"
-    }
-  ],
-  "milestones": [
-    {
-      "title": "Vritant App",
-      "date": "Nov 2022",
-      "icon": "FaGlobe",
-      "color": "white",
-      "description": "Currenlty working on this amazing app.",
-      "url": "https://github.com/bhattabhi013/Vritant"
-    },
-    {
-      "title": "Started Developing and open-source",
-      "date": "March 2022",
-      "icon": "FaGithub",
-      "color": "black",
-      "description": "One of my best and life changing decision ever.",
-      "url": "https://github.com/Ansh-Rathod"
-    },
-    {
-      "title": "Launched first mobile application.",
-      "date": "May 2021",
-      "icon": "FaAndroid",
-      "color": "green",
-      "description": "Launched my first mobile application on playstore.",
-      "url": "https://play.google.com/store/apps/details?id=com.plasma.covid"
-    }
-  ]
-}
+    "name": "Abhishek Bhatt",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full-Stack developer | Flutter | NodeJs | Angular | React",
+    "tags": ["Flutter", "Node,js", "Angular", "Javascript", "Typescript", "open-source"],
+    "socials": [
+      { "platform": "twitter", "url": "https://twitter.com/AbhishekBtwt" },
+      { "platform": "github", "url": "https://github.com/bhattabhi013" },
+      { "platform": "instagram", "url": "https://www.instagram.com/bwith_abhi" }
```

Should be `icon`. Not sure `platform` is valid?
```suggestion
      { "platform": "FaTwitter", "url": "https://twitter.com/AbhishekBtwt" },
      { "platform": "FaGithub", "url": "https://github.com/bhattabhi013" },
      { "platform": "FaInstagram", "url": "https://www.instagram.com/bwith_abhi" }
```